Build/MQTTnet.nuspec

```diff
@@ -16,6 +16,8 @@
 * [Core] Decreased object allocations (#1324, thanks to @gfoidl).
 * [Core] Decreased object allocations when logging is not active (thanks to @gfoidl, @Tymoniden).
 * [Client] Fixed issue in _MqttApplicationMessageBuilder.WithPayload_ (#1322, thanks to @gfoidl).
+* [Client] Adjusted default SslProtocol values to Tls12 and Tls13 (#1347).
+* [Extensions.WebSocket4Net] Adjusted default SslProtocol values to Tls12 and Tls13 (#1347).
```

Comment:
Please mention all other changes made in this PR.

Reply:
* Fixed a deadlock in AsyncQueue
* Fixed an issue in MqttClientConnection where a message could be dequeued and lost if the token is cancelled
* Fixes for AsyncQueue so the semaphore count aligns with queue count

BTW the checks for token cancellation exist in the latest as well. Those should be fixed as well. Basically, the cancellation check should take place before dequeuing, or the message should be requeued on cancellation.

Reply:
I mean to mention this in the release notes.